module-check/src/main/clojure/clojure/core/typed/base_env.clj

```diff
@@ -1443,7 +1443,17 @@ clojure.core/rand-nth (All [x] [(U (Indexed x) (SequentialSeqable x)) -> x])
   this-ns
 clojure.set/union (All [x] [(Set x) * -> (Set x)])
 clojure.set/intersection (All [x] [(Set x) (Set x) * -> (Set x)])
+
 clojure.set/difference (All [x] [(Set x) (Set Any) * -> (Set x)])
+clojure.set/subset? [(Set Any) (Set Any) -> Boolean]
+clojure.set/superset? [(Set Any) (Set Any) -> Boolean]
+clojure.set/join [(Set (Map Any Any)) (Set (Map Any Any)) -> (Set (Map Any Any))]
+clojure.set/index [(Set (Map Any Any)) (Vec Any) -> (Map (Map Any Any) (Set (Map Any Any)))]
+clojure.set/map-invert (All [a b] [(Map a b) -> (Map b a)])
+clojure.set/project [(Set (Map Any Any)) (Vec Any) -> (Set (Map Any Any))]
+clojure.set/rename [(Set Any) (Map Any Any) -> (Set Any)]
```

Comment:
Not sure I agree. Example
```clojure
(rename #{{:one 1} {:two 2}} {:one "one" :two "two"}) => #{{"one" 1} {"two" 2}}
```
Type of keys is changed from keywords to strings. And in proposed annotation type of initial and result maps is the same (`a`). I think it should rather be 
```clojure
(All [a b c] [(Set (Map a b)) (Map Any c) -> (Set (Map c b))])
```